src/dashboards/utils/cellGetters.ts

```diff
@@ -34,7 +45,8 @@ export const isCellUntitled = (cellName: string): boolean => {
 export const getNewDashboardCell = (
   state: AppState,
   dashboard: Dashboard,
-  clonedCell?: Cell
+  clonedCell: Cell,
```

Comment:
this gives me a funny feeling since clonedCell is an optional parameter that we're making required, but is still optional to the parent

Reply:
There's only 1 instance of `getNewDashboardCell` being called and it checks if clonedCell is truthy or not. This PR changes `getNewDashboardCell` so it seemed fine to adjust the parameters.

The parent has multiple calls and i didn't want to mess with the parent code in this PR.

Reply:
for sure, I guess maybe we add a default there since the parent of `getNewDashboardCell` is `createCellWithView` which accepts an optional `clonedCell` parameter. I guess just a nit since `clonedCell` can be `undefined`?

Reply:
https://github.com/influxdata/ui/blob/master/src/cells/actions/thunks.ts#L73

Reply:
I don't know, my brain gets foggy at this hour so I might be super paranoid

Reply:
Hmm, i don't think it needs a default. i don't want to inadvertently change anything unrelated.